packages/docs/src/components/DocsExample.vue

```diff
@@ -21,12 +21,12 @@
         />
         <DocsCode
           language="markup"
-          :code="parsed.template"
+          :code="applyTranslations(parsed.template)"
```

Comment:
we shoudn't apply `applyTranslations` here, because we applied it in component `DocsCode`
`:code="parsed.template"`